sys-block/arcconf/arcconf-2.01.22270.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Could you please split those two bumps into separate commits, please?

Reply:
Done.